integration-cli/docker_cli_authz_plugin_v2_test.go

```diff
@@ -72,7 +75,7 @@ func (s *DockerAuthzV2Suite) TestAuthZPluginAllowNonVolumeRequest(c *check.C) {
 
 	out, err = s.d.Cmd("ps")
 	c.Assert(err, check.IsNil)
-	c.Assert(assertContainerList(out, []string{id}), check.Equals, true)
+	c.Assert(assertContainerList(RemoveOutputForExistingElements(out, existingContainers), []string{id}), check.Equals, true)
```

Comment:
I don't really understand why this test should be looking at the container list. Is the intent to verify the container is actually running? I think the successful exit of `docker run`should already tell us.

I'd rather remove this check than compare against "existing elements".

cc @riyazdf: for context, we want to run these tests against different engines, some of which might have already running containers, so the output of `ps` is undefined.